integrations/access/opsgenie/app.go

```diff
@@ -181,7 +200,19 @@ func (a *App) checkTeleportVersion(ctx context.Context) (proto.PingResponse, err
 	return pong, trace.Wrap(err)
 }
 
+// onWatcherEvent is called for every cluster Event. It will filter out non-access-request events and
+// call onPendingRequest, onResolvedRequest and on DeletedRequest depending on the event.
```

Comment:
Where does it call `onPendingRequest`, `onResolvedRequest` and `on DeletedRequest` (sic)? I don't see these calls.